docs/user/alerting/troubleshooting/event-log-index.asciidoc

```diff
@@ -201,3 +201,63 @@ Here’s an example of what failed credentials from Google SMTP might look like
 535 5.7.8  https://support.google.com/mail/?p=BadCredentials e207sm3359731pfh.171 - gsmtp"""
 },
 --------------------------------------------------
+
+[float]
+==== Most Expensive Rules
+
+Usually, the most expensive Rules also have the highest `event.duration`. You can check 
+top expensive rules by loading https://gist.github.com/pmuellr/e9ac5cda106589dc52477ede9accb20c[this dashboard] or running this query:
```

Comment:
I think we probably shouldn't include the link to the dashboard, unless we can find some Elastic-owned site to contain it.  And we've had some feedback that the dashboard is too "framework engineering" focused (for us) vs customer focused.  Very likely customers will be handy with the dev tools script, and it does do what it says on the tin, so ... I think it's probably good enough ...